server/src/services/patient_service.ts

```diff
@@ -39,4 +39,8 @@ export class PatientService {
 
         await this.statusRepository.updatePatientStatusFields(patientId, fields);
     }
+
+    async getPatientStatusFields(patientId: number): Promise<StatusFields> {
```

Comment:
Can you update line 35 to call this method instead of the repository so theres a single place we access this domain object